src/modules/digits/converters/fa.ts

```diff
@@ -0,0 +1,34 @@
+import { enNums, faNums, arNums } from "./numsList";
+import strConverter from "./strConverter";
+
+type DigitsFaToX = (value: string) => string;
+
+/**
+ * digitsFaToEn
+ *
+ * @category Digits
+ * @description Takes a string and
+ * returns a string that represents the same value but
+ * its farsi digits are replaced with english digits
+ */
+const digitsFaToEn: DigitsFaToX = (value) => {
+	if (typeof value !== "string") throw TypeError("PersianTools: digitsFaToEn - The input must be string");
+
+	return strConverter({ str: String(value), originCharList: faNums, destCharList: enNums });
+};
+
+/**
+ * digitsFaToAr
+ *
+ * @category Digits
+ * @description Takes a string and
+ * returns a string that represents the same value but
+ * its farsi digits are replaced with arabic digits
+ */
+const digitsFaToAr: DigitsFaToX = (value) => {
```

Comment:
```DigitsFaToX``` doesn't make any sense. please use the right Interface name instead like:

```ts
type DigitsConverter<T = string, R = string> = (value: T) => R;
```
`T` is the value type that can be dynamic and `R` is the result type which is dynamic too. This approach helps you to have a global type and use it everywhere you want such as:
```typescript
const digitsFaToAr: DigitsConverter = (value) => {
```
or for `digitsFaToEn`
```typescript
const digitsFaToEn: DigitsConverter<string, number> = (value) => {
```
or in `digitsEnToFa` case:
```typescript
const digitsEnToFa: DigitsConverter<number, string> = (value) => {
```